eas.json

```diff
@@ -4,7 +4,7 @@
   },
   "build": {
     "production": {
-      "node": "16.13.0",
+      "node": "v18.19.1",
       "yarn": "1.22.19",
```

Comment:
Should we upgrade the node version on our computer for this update?

Reply:
yes

Reply:
Can you please briefly explain how to do this in PR?

Reply:
done

Reply:
or with [asdf](https://asdf-vm.com/): `asdf install nodejs 18.19.1`